<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommodityLoginDlg</class>
 <widget class="QDialog" name="CommodityLoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信息登记系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/store.jpg</normaloff>:/images/store.jpg</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_Sort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>排序</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_Key">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>编号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>名称</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>价格</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Order">
      <item>
       <property name="text">
        <string>升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>降序</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Sort">
      <property name="text">
       <string>排序</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>商品信息</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>...</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Information">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>信息</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_Id">
          <property name="text">
           <string>编号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Id"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_Name">
          <property name="text">
           <string>名称：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_Price">
          <property name="text">
           <string>价格：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Price"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_Operation">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_Insert">
      <property name="text">
       <string>插入数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Search">
      <property name="text">
       <string>查询数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Update">
      <property name="text">
       <string>更新数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Delete">
      <property name="text">
       <string>删除数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
